Fakes/OpenActive.FakeDatabase.NET/FakeBookingSystem.cs

```diff
@@ -358,7 +358,7 @@ public static (ReserveOrderItemsResult, long?, long?) LeaseOrderItemsForFacility
         }
 
         // TODO this should reuse code of LeaseOrderItemsForClassOccurrence
-        public static (ReserveOrderItemsResult, List<long>) BookOrderItemsForClassOccurrence(
+        public static (ReserveOrderItemsResult, List<long>, List<string>) BookOrderItemsForClassOccurrence(
```

Comment:
Noting that this is also going to require imageUrls and barcodes to be generated, should we create struct and return a list<> of these together, instead needing them to be zipped?

We can then bundle the OrderItemId into the same struct?

Reply:
I agree, it makes more sense and it will be more clear :)